macros/miscs/debugging.cfg

```diff
@@ -35,3 +35,19 @@ gcode:
             {% endfor %}
         {% endif %} 
     {% endfor %}
+
+# Specific macro to raise an error during the print_start and stop in case of ERCF error
+[gcode_macro _DO_ERROR_ERCF_PAUSE]
+gcode:
+    {% set verbose = printer["gcode_macro _USER_VARIABLES"].verbose %}
+    {% set INITIAL_TOOL = printer["gcode_macro START_PRINT"].initial_tool %}
+
+    {% if printer.ercf.enabled|int == 1 %}
+        {% if printer.ercf.tool|int != INITIAL_TOOL or printer.ercf.filament != "Loaded" %}
+            {action_raise_error("ERROR in loading filament from ERCF !!! Check and restart the print.")}
+        {% else %}
+            {% if verbose %}
+                RESPOND MSG="T{INITIAL_TOOL} ready"
+            {% endif %}
+        {% endif %}
+    {% endif %}
```

Comment:
Can you add a free new line at the end of the file?
Also I feel that this macro definition should be moved directly into the `ercf.cfg` file as it's used only when there is an ERCF connected.